tools/docker/README.md

```diff
@@ -8,7 +8,7 @@
 **Build container**
 
 * `cd ~/LibreELEC`
-* `docker build --pull -t libreelec tools/docker/bionic`
+* `docker build --pull -t libreelec tools/docker/eoan`
```

Comment:
```suggestion
* `docker build --pull -t libreelec tools/docker/focal`
```

Reply:
19.10 is basically EOL already, 20.04 is LTS

Reply:
Thanks, I got the ubuntu names mixed up